src/lib/cache.ts

```diff
@@ -125,7 +127,7 @@ function _get<T>(key) {
   })
 }
 
-function _set(key, data) {
+function _set(key, data, options: APIOptions) {
```

Comment:
The API options really don't make sense to pass thru here, to the cache. We really just wanted to access the `requestThrottleMs` option. I can adjust to pass that thru as more appropriate cache options.